modules/persistent-storage-csi-vsphere-install-issues.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+//
+
+[id="persistent-storage-csi-vsphere-install-issues_{context}"]
+= Removing an upstream vSphere CSI Operator Driver
+
+If you have installed an upstream vSphere Container Storage Interface (CSI) Operator Driver, and then try to update your cluster to {product-title} 4.10 or later, {product-title} prevents the update.
+
+To correct this issue and update the cluster, a cluster admin must:
+
+. Delete the upstream vSphere CSI driver Deployment object.
+. Delete the upstream vSphereCSI driver Daemonset objects:
+.. Be sure to delete `CSIDriver`:
++
+[output, screen]
+----
+~ $ oc delete CSIDriver csi.vsphere.vmware.com
```

Comment:
For the command and output, we'd better unify with the same style like : https://deploy-preview-40571--osdocs.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-vsphere.html#csi-tp-enable_persistent-storage-csi-vsphere
Divided the command and output to 2 different code blocks for customer copy conveniently.